CHANGELOG.rst

```diff
@@ -21,6 +21,14 @@ with advance notice in the **Deprecations** section of releases.
 pytest 4.6.0 (2019-05-31)
 =========================
 
+Important
```

Comment:
In retrospect, I could have done this myself. 😅

Reply:
hehehe -- I did it anyway all good!